core/src/scss/utilities/mixins/link/_link.scss

```diff
@@ -9,14 +9,21 @@
 /// @param {variable} $hover - Default hover color
 ///
 /// @group mixin
-@mixin link($color: color(link), $hover: color(link--hover)) {
+@mixin link($color: color(link), $hover: color(link--hover), $focus: color(focus)) {
+  @include padding(null 0.15rem);
   color: $color;
-  text-decoration: underline;
   font-weight: $su-font-semi-bold;
+  text-decoration: underline;
 
   &:hover,
   &:focus,
   &:active {
     color: $hover;
   }
+
+  &:focus {
+    box-shadow: $su-button-stroke $focus, $su-focus-shadow;
```

Comment:
@yvonnetangsu, I want to use the values of $su-button-stroke, but that naming of that variable might not be appropriate in this file. Your thoughts?

Reply:
@cjwest I'm not super concerned with this one myself. Others might disagree with me though 😃